### PR TITLE
build/ant: create java skeleton project which is using ant

### PR DESCRIPTION
**Pull request by slavidichkov**
TODO: Add description for your task.

### CODE REVIEW

File: ant/tests/com/clouway/MyProjectTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Slavi Dichkov (slavidichkof@gmail.com)
+ */
+public class MyProjectTest {
+    @Test
+    public void probaTest() {
+       assertEquals(2,2);
```
Comment (tsony-tsonev) [inline ant/tests/com/clouway/MyProjectTest.java]: and again..... do not commit stuff you don't need

Comment (slavidichkov) [inline ant/tests/com/clouway/MyProjectTest.java]: in my ant project i have to use junit library and I needed this test

Comment (slavidichkov) [inline ant/tests/com/clouway/MyProjectTest.java]: i will fix its name

Comment (slavidichkov) [inline ant/tests/com/clouway/MyProjectTest.java]: fixed

File: ant/build.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    </path>
+
+    <pathconvert property="mf.classpath" pathsep=" ">
+        <path refid="build.classpath"/>
+        <flattenmapper/>
+    </pathconvert>
+
+    <target name="jarjarpackage" depends="compile">
+        <copy todir="${build}" includeemptydirs="false">
+            <fileset dir="" includes="**/*.properties" excludes="out/"/>
```
Comment (tsony-tsonev) [inline ant/build.xml]: put 'out/' dir in variable

File: ant/build.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            <manifest>
+                <attribute name="Class-Path" value="${mf.classpath}"/>
+                <attribute name="${main_class_name}" value="com.clouway.MyProject"/>
+            </manifest>
+        </jarjar>
+    </target>
+
+    <target name="jarpackage" depends="compile">
+        <copy todir="${build}" includeemptydirs="false">
+            <fileset dir="" includes="**/*.properties" excludes="out/"/>
```
Comment (tsony-tsonev) [inline ant/build.xml]: same here

File: ant/build.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        </copy>
+        <jar destfile="${build}/MyProject.jar" basedir="${build}">
```
Comment (tsony-tsonev) [inline ant/build.xml]: you have jarname variable define you can use it here

File: ant/build.xml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+            <zipgroupfileset dir="lib" includes="*.jar" excludes="jarjar-1.4.jar"/>
+            <manifest>
+                <attribute name="Class-Path" value="${mf.classpath}"/>
+                <attribute name="${main_class_name}" value="com.clouway.MyProject"/>
+            </manifest>
+        </jar>
+    </target>
+
+    <target name="run">
+        <java jar="${build}/MyProject.jar" fork="true"/>
```
Comment (tsony-tsonev) [inline ant/build.xml]: jarname variable should be used here also

**Comment by tsony-tsonev** [comment]
LGTM you can merge this PR after you fix the latest comments

File: ant/build.xml (hunk 2 above)
Comment (slavidichkov) [inline ant/build.xml]: fixed

File: ant/build.xml (hunk 3 above)
Comment (slavidichkov) [inline ant/build.xml]: fixed

File: ant/build.xml (hunk 4 above)
Comment (slavidichkov) [inline ant/build.xml]: fixed

File: ant/build.xml (hunk 5 above)
Comment (slavidichkov) [inline ant/build.xml]: fixed

**Pull request merged**